containers/cpp-mariadb/.vscode/tasks.json

```diff
@@ -0,0 +1,16 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
```

Comment:
not valid json ... hence the red syntax highlighting

Reply:
The invalid JSON is part of the original container. I don't think this repo makes a differentiation between JSON and JSON With Comments.